app/assets/stylesheets/application.css

```diff
@@ -14,27 +14,11 @@
  *= require_self
```

Comment:
- [ ] Kindly don't use the same branch for each milestone because this is against the workflow :fearful: It would be very nice if you submit that pull request which has your controller specs or if you don't have that then kindly make a new branch and include your controller specs in that branch and then make a pull request and submit it for review to follow the Gitflow :+1: